WWCalendarTimeSelectorDemo/ViewController.swift

```diff
@@ -130,15 +141,22 @@ class ViewController: UIViewController, UITableViewDelegate, UITableViewDataSour
         cell.textLabel?.text = demo[(indexPath as NSIndexPath).row]
         return cell
     }
+}
+
+// MARK: - WWCalendarTimeSelectorProtocol
+
+extension ViewController: WWCalendarTimeSelectorProtocol {
     
-    func WWCalendarTimeSelectorDone(_ selector: WWCalendarTimeSelector, date: Date) {
+    func WWCalendarTimeSelectorDone(_ selector: WWCalendarTimeSelector, date: Date, selectedOption: String?) {
         print("Selected \n\(date)\n---")
+        print("Selected option \(selectedOption ?? "nil")")
```

Comment:
we should not have print in release version

Reply:
this is the example app that comes with the library, it's not for production